autoPyTorch/pipeline/components/training/trainer/base_trainer_choice.py

```diff
@@ -98,11 +98,11 @@ def get_components(self) -> Dict[str, autoPyTorchComponent]:
         return components
 
     def get_hyperparameter_search_space(
-        self,
-        dataset_properties: Optional[Dict[str, str]] = None,
-        default: Optional[str] = None,
-        include: Optional[List[str]] = None,
-        exclude: Optional[List[str]] = None,
+            self,
+            dataset_properties: Optional[Dict[str, str]] = None,
```

Comment:
I think these are not your changes, could you look into them?

Reply:
Ok, I will check all the strange changes later (tomorrow). I think this came from the merge to older `refactor_development` branch. Once I updated the branch, I haven't got the changes.